### PR TITLE
Fix/probesize

### PR DESCRIPTION
**Pull request by thomasjohansen**
Tested https://github.com/jellyfin/jellyfin/pull/10135 and could not get it to work.

This works for me.

### CODE REVIEW

File: MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs [hunk 1]
```diff
@@ -429,12 +431,22 @@ public Task<MediaInfo> GetMediaInfo(MediaInfoRequest request, CancellationToken
                 analyzeDuration = "-analyzeduration " + ffmpegAnalyzeDuration;
             }
 
+            if (analyzeDuration.Length > 0)
```
Comment (JPVenson) [inline MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs:434]: check for empty or set analyzeDuration to null

**Review by JPVenson — CHANGES_REQUESTED** [review]

File: MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs (hunk 1 above)
Comment (thomasjohansen) [reply MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs:434]: I will use the string.IsNullOrEmpty as above. Not sure why this changes anything.

Comment (JPVenson) [reply MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs]: code style

**Review by JPVenson — APPROVED** [review]

**Review by Shadowghost — APPROVED** [review]
It would be nice if you can squash the commits.

**Comment by thomasjohansen** [comment]
Is that not possible when you merge it? At least that is what I see here:
https://github.blog/2016-04-01-squash-your-commits/

**Comment by JPVenson** [comment]
FYI: 
https://jellyfin.org/docs/general/contributing/development#pull-request-guidelines

**Comment by JPVenson** [comment]
> > FYI: https://jellyfin.org/docs/general/contributing/development#pull-request-guidelines
> 
> I'm sorry about the force push. My forked repo is not clean and this was the only way I could figure out how to do it.

No worries. That was more related to the "Please make speaking commit messages" part of the guidelines :-D

**Comment by thomasjohansen** [comment]
> > > FYI: https://jellyfin.org/docs/general/contributing/development#pull-request-guidelines
> > 
> > 
> > I'm sorry about the force push. My forked repo is not clean and this was the only way I could figure out how to do it.
> 
> No worries. That was more related to the "Please make speaking commit messages" part of the guidelines :-D

Ah! Hope you will approve anyway :)

**Comment by thomasjohansen** [comment]
@JPVenson will you approve it so it can be merged?

**Review by cvium — APPROVED** [review]

**Comment by Shadowghost** [comment]
It would be nice if you can rename the commit to something more descriptive and force push before we merge

**Pull request merged**